packages/affiliates/libs/processors.js

```diff
@@ -162,6 +162,9 @@ function EmpBalances(handlers = {}, { collateral, tokens } = {}) {
     LiquidationDisputed(/* caller, originalExpirationTimestamp, shutdownTimestamp*/) {
       // nothing
     },
+    DisputeSettled() {
```

Comment:
this was such a tiny change i included it in rewards. it does not change output at all.